db/schema.rb

```diff
@@ -17,24 +16,30 @@
   enable_extension "plpgsql"
 
   create_table "users", force: :cascade do |t|
-    t.string   "email",                  limit: 255, default: "", null: false
```

Comment:
Limit constraints disappeared, because of using newer version of Postgresql (10.5)